api/handlers/app.go

```diff
@@ -606,6 +607,33 @@ func (h *App) getSSHEnabled(r *http.Request) (*routing.Response, error) {
 	}), nil
 }
 
+func (h *App) getAppFeature(r *http.Request) (*routing.Response, error) {
+	featureName := routing.URLParam(r, "name")
+	switch featureName {
+	case "ssh":
+		return routing.NewResponse(http.StatusOK).WithBody(map[string]any{
+			"name":        "ssh",
+			"description": "Enable SSHing into the app.",
+			"enabled":     false,
+		}), nil
+	case "revisions":
+		return routing.NewResponse(http.StatusOK).WithBody(map[string]any{
+			"name":        "revisions",
+			"description": "Enable versioning of an application",
+			"enabled":     false,
+		}), nil
+	default:
+		return routing.NewResponse(http.StatusOK).WithBody(map[string]any{
```

Comment:
I guess we should return some sort of an HTTP error code here instead of OK. Maybe the same one as on CF for VMs

Reply:
agreed (totally missed that). Updated the PR accordingly.